build.gradle

```diff
@@ -12,6 +12,26 @@ plugins {
     id "groovy"
 }
 
+final String compileVersion = project.hasProperty("toolchainCompileVersion") ?
+        (project.property("toolchainCompileVersion") as String).replaceAll("^([0-9]+).*", "\$1") : "8"
+final String testVersion = project.hasProperty("toolchainTestVersion") ?
+        (project.property("toolchainTestVersion") as String).replaceAll("^([0-9]+).*", "\$1") : "8"
+final String javadocVersion = project.hasProperty("toolchainJavadocVersion") ?
+        (project.property("toolchainJavadocVersion") as String).replaceAll("^([0-9]+).*", "\$1"): "8"
+def toolchainVm = project.property("toolchainVm")
+JvmImplementation vmImplementation = null
+if ("hotspot" == toolchainVm) {
+    vmImplementation = JvmImplementation.VENDOR_SPECIFIC
```

Comment:
There is no JvmImplementation.HOTSPOT

Probably works with Azul Platform Prime Jars, but I do not have those x)